bundle/Configuration/ScheduledVisibilityConfiguration.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Configuration;
+
+use function in_array;
+
+final class ScheduledVisibilityConfiguration
+{
+    public function __construct(
+        private readonly string $handler,
+        private readonly bool $enabled,
+        private readonly bool $allContentTypes,
+        private readonly array $allowedContentTypes,
+    ) {}
+
+    public function isEnabled(): bool
+    {
+        return $this->enabled;
+    }
+
+    public function isAllContentTypes(): bool
+    {
+        return $this->allContentTypes;
+    }
+
+    public function getAllowedContentTypes(): array
+    {
+        return $this->allowedContentTypes;
+    }
+
+    public function getHandler(): string
```

Comment:
Rename to `getHandlerIdentifier()`.

Reply:
fixed in 45da079a81f441d535f98ed9ef6ed2b14c0f6392